api/jobs/api.py

```diff
@@ -230,6 +235,12 @@ def list(self, request):
         smart_socs = [soc for soc in available_socs
                       if soc.get("soc_code") in smart_soc_codes]
 
+        # Order response in by the highest fuzzy match score
+        smart_socs = sorted(smart_socs,
+                            key=lambda k: k.get("input_match_score"),
```

Comment:
Here we sort everything based on the input_match_score, even though O*Net results may be included based on related terms and not actually contain the search term in the title. So I wonder if it makes sense to preserve the ordering from O*Net and present the results as [fuzzy-matched results, O*Net results]. What do you think?

Reply:
Makes sense to me! Thanks for the feedback. Changed so 
* fuzzy-matched results are returned in best-worst match order (if the SOC codes aren't present from O*NET)
* O*NET results are appended to the end of that, with order preserved
* If no keyword is given (None, '', etc.), the full list of transitions is returned as a default